main/src/ca/uwaterloo/flix/api/lsp/provider/completion/Completion.scala

```diff
@@ -130,15 +133,26 @@ object Completion {
                                    insertTextFormat: InsertTextFormat) extends Completion
 
   /**
-    * Represents a Type completion (enums, aliases)
+    * Represents an EnumType completion
+    *
+    * @param name          the name of the EnumType.
+    * @param priority      the priority of the EnumType.
+    * @param textEdit      the edit which is applied to a document when selecting this completion.
+    * @param documentation a human-readable string that represents a doc-comment.
+    */
+  case class EnumTypeCompletion(name: String, priority: String, textEdit: TextEdit,
+                                documentation: Option[String]) extends Completion
```

Comment:
Not sure if we want to store the TypedAst?

Reply:
We could store the EnumSym

Reply:
Don't know if this is better, or it's to many arguments?